### PR TITLE
Blog - Working with GitHub Actions Environment Variables and Secrets

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL: https://641ae7fd0df1570087554ee9--ephemeral-narwhal-3ac3b2.netlify.app/blog/github-actions-environment-variables-and-secrets/

File: blog/_posts/2023-05-02-github-actions-environment-variables-and-secrets.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+Like most CI/CD platforms, [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions), a popular CI/CD platform, offers a way for you to manage your CI/CD environment variables and secrets. In this article, you'll learn how GitHub Actions work, when you should use them, and how to get started.
+
+To follow along, you'll need a [GitHub](/blog/ci-comparison) account to fork the repo and try out GitHub Actions. You'll also need some familiarity with YAML, the standard language for writing and managing [GitHub Actions](/blog/continuous-integration) configuration files.
+
+![Guide]({{site.images}}{{page.slug}}/guide.png)\
+
+## What Are GitHub Actions Environment Variables and Secrets
```
Comment (balapriyac) [inline blog/_posts/2023-05-02-github-actions-environment-variables-and-secrets.md:23]:
```suggestion
## What Are GitHub Actions Environment Variables and Secrets?
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Looks great!

**Comment by balapriyac** [comment]
@jalletto this is ready to merge, too.

**Review by jalletto — APPROVED** [review]

**Pull request merged**